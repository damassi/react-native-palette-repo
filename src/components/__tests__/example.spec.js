/* eslint-disable */
import React from 'react';
import styled from 'styled-components/native';
import renderer from 'react-test-renderer';
// import { Theme, Box } from '@artsy/palette';

const App = styled.View``;
const Text = styled.Text`
  color: black;
`;

test('example', () => {
  const tree = renderer
    .create(
      <App>
        <Text>hi</Text>
      </App>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
