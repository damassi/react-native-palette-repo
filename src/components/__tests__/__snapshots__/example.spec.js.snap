// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example 1`] = `
<View
  style={
    Array [
      Object {},
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "black",
        },
      ]
    }
  >
    hi
  </Text>
</View>
`;
